imlac/imlac_cpu.c

```diff
@@ -602,13 +605,13 @@ flag_check (uint16 flag)
 }
 
 static uint16
-irq_iot (uint16 insn, uint16 AC)
+irq_iot (uint16 insn, uint16 CUR_AC)

```

Comment:
Please remove these changes.

Reply:
Shadows AC -- is shadowing AC intentional?

Reply:
Yes.  The global AC should *not* be used inside this function.